src/libraries/System.Private.CoreLib/src/System/Char.cs

```diff
@@ -72,6 +72,14 @@ namespace System
         private static bool IsLatin1(char c) => (uint)c < (uint)Latin1CharInfo.Length;
 
         // Return true for all characters below or equal U+007f, which is ASCII.
+
+        /// <summary>
+        /// Returns <see langword="true"/> iff <paramref name="c"/> is an ASCII
```

Comment:
```suggestion
        /// Returns <see langword="true"/> if <paramref name="c"/> is an ASCII
```
I don't think docs really use `iff`...